lib/test.gi

```diff
@@ -584,7 +584,7 @@ InstallGlobalFunction( "TestDirectory", function(arg)
       g := GASMAN_STATS();    
       return g[1][8] + g[2][8];    
   end;
-        
+          
```

Comment:
Also this line can go. And I see that your first commit adds stuff here, and then some other commit removes it again. The first commit (and its message) are now almost pointless. Perhaps squash everything together? Or perhaps you want the change in `lib/string.gi` to be in a separate commit?

You could do this:
```sh
git checkout THIS-BRANCH
git rebase master # to make sure you really sit on master
git reset HEAD # discard all commits, but keep their changes

# now group changes into fresh commits:
git commit -m "Validate argument to StringOfMemoryAmount" lib/string.gi

# rest can probably be in a single big commit?
```